### DISCUSSION TITLE
Institute email recognition failed

### DISCUSSION THREAD

**Discussion by naked-head** [community/community#51380] [GitHub Education]
Hi,
I'm a teacher and I'm trying to request a discount to get github benefits as a teacher. To do that I want to use my institute email address @sarrocchi.it but when I select this mail I get this warning message:

_We require applicants of Istituto di Istruzione Superiore Tito Sarrocchi to use one of these school-issued email addresses to apply:_
_- students.sarrocchi.it_
_Please select a different email address._

and I can't do anything else.
The emails with domain students.sarrocchi.it are for students but all the teacher are under domain sarrocchi.it (without the students part). I know many colleagues who signed up with the teacher's domain and many students with their own domain, it seems something changed in the github policy and now I can't get the subscription anymore.

How can I get autorized with my institute mail?

**Comment by F0XN0T** [comment]
I apologize, but it seems that your request for a discount for the GitHub benefits as a teacher is outside the scope of the discussion prompt you provided. However, I understand that you are encountering difficulties with accessing the GitHub Student Developer Pack using your institute email address. It's frustrating to receive a warning message when selecting your email address, and it's understandable that you are seeking a solution.

One possible solution could be to contact GitHub's customer support team and explain the situation to them. They may be able to assist you in verifying your identity as a teacher and provide guidance on how to proceed. You could also explore alternative email addresses that are accepted by GitHub, such as a personal email address.

It's important for platforms like GitHub to be responsive to their users' needs, particularly in the Education section. I hope that you are able to find a solution that works for you, and that GitHub can improve its policies and procedures to better support its student and teacher consumers.

**Reply by naked-head** [reply]
Thanks for your reply but before posting here I wrote to the support team and open a ticket , they closed my ticket and replied to ask to the community....

That's very very very frustrating...

I'm going to ask my school's mail admin to create me a student mail with the only domain recognized by GitHub, so I can get around their policy
